compiler/rustc_passes/src/check_attr.rs

```diff
@@ -619,7 +619,30 @@ impl CheckAttrVisitor<'_> {
             }
             // we check the validity of params elsewhere
             Target::Param => return false,
-            _ => None,
+            Target::Expression | Target::Statement | Target::Arm => Some(target.name()),
```

Comment:
`ForeignModule` can go in this arm too. The name `foreign module` is better than `extern block`.

Reply:
Resolved by 2be0094